sdk/FEATURES.md

```diff
@@ -22,6 +22,7 @@ and test pipelines found in the KFP repository.
     - [Any Sequencer](#any-sequencer)
     - [Tekton Pipeline Variables](#tekton-pipeline-variables)
     - [Sidecars](#sidecars)
+    - [Caching](#caching)
```

Comment:
KFP caching is not a native Tekton implementation, can you move this to the `Pipeline DSL Features with a Custom Tekton Implementation` section? thanks.